balboa-http/src/main/resources/config/config.properties

```diff
@@ -26,8 +26,8 @@ balboa.datastore = cassandra
 memcached.servers = localhost:11211
 
 # Cassandra servers. Servers should be separated by a comma.
-cassandra.servers = localhost:9160
-cassandra.keyspace = Metrics2012
+cassandra.servers = localhost:9042
+cassandra.keyspace = "Metrics2012"
```

Comment:
Could you stick a comment in here that explains the necessity of this? Keeping in mind that not only the future person modifying this code will read it, but also other people responsible for converting from Astyanax drivers to Datastax drivers will see it and need to apply this to their code base.